homeassistant/components/elmax/translations/en.json

```diff
@@ -1,13 +1,15 @@
 {
     "config": {
         "abort": {
-            "already_configured": "Device is already configured"
+            "already_configured": "Device is already configured",
```

Comment:
File should be removed

Reply:
Done in 5102a0de8335067beb912f9dad6c18ad78db64e9, thanks!